src/main/kotlin/io/github/karmasmp/karma/command/CreatorCommand.kt

```diff
@@ -0,0 +1,20 @@
+package io.github.karmasmp.karma.command
+
+import io.github.karmasmp.karma.player.creator.Creator
+import io.papermc.paper.command.brigadier.CommandSourceStack
+
+import org.bukkit.entity.Player
+
+import org.incendo.cloud.annotations.Command
+import org.incendo.cloud.annotations.processing.CommandContainer
+
+@Suppress("unused", "unstableApiUsage")
+@CommandContainer
+class CreatorCommand {
+    @Command("creatormode")
+    fun echo(css: CommandSourceStack) {
+        if(css.sender is Player) {
```

Comment:
Something I believe I saw mentioned but didn't get to trying is I think you can just substitute CommandSourceStack with Player in the arguments and that _should_ work (but then means its not a console command)

Reply:
Applies to several other places too